tests/cases/fourslash/refactorConvertToEs6Module_export_named.ts

```diff
@@ -3,7 +3,7 @@
 // @allowJs: true
 
 // @Filename: /a.js
-////[|exports.f = function() {}|];
+////[|exports.f|] = function() {};
 ////exports.C = class {};
```

Comment:
Do we only offer the fix on the first one?

Reply:
Yes, only at the single location that comes from `commonJsModuleIndicator`.